runtime/onert/core/include/backend/IPortableTensor.h

```diff
@@ -41,17 +41,31 @@ class IPortableTensor : public ITensor
   IPortableTensor(const ir::OperandInfo &info) : _info(info) {}
 
   virtual ~IPortableTensor();
-  virtual const ir::Sparsity *sparsity() const { return nullptr; }
+
+public:
+  // It is introduced to reduce virtual method call overhead on inference
+  // So derived class should maintain actual OperandInfo to "_info" field
+  // Ex. CPU backend getShape() in OperationUtils.h is called frequently on inference
+  //     and it calls get_info()
   const ir::OperandInfo &get_info() const { return _info; }
-  float data_scale() const override { return _info.typeInfo().scale(); }
-  int32_t data_zero_point() const override { return _info.typeInfo().zero_point(); }
-  const std::vector<float> &data_scales() const override { return _info.typeInfo().scales(); }
+  const ir::Sparsity *sparsity() const { return _info.typeInfo().sparsity(); }
+
+  // Finailized methods for IPortableTensor by "_info" field read
```

Comment:
```suggestion
  // Finalized methods for IPortableTensor by "_info" field read
```

typo.

Or we may remove this comment since `final` keyword says the same thing.